payroll-processor-functions/src/payroll-processor-functions/payroll-processor-functions.csproj

```diff
@@ -4,7 +4,15 @@
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
     <RootNamespace>Payroll.Processor.Functions</RootNamespace>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
```

Comment:
This turns on C#8 nullable type checking.

Reply:
Can we use the `<LanguageVersion>latest</LanguageVersion>` to bring in all C# 8 features?

Reply:
I don't think that's enough - we have to opt-in.

https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references#nullable-contexts

Also, I think the property is `<LangVersion>...</LangVersion>`

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#edit-the-project-file

Reply:
Yeah it's LangVersion, was recalling from memory. Good to know!